Libraries/Text/RCTTextView.m

```diff
@@ -127,6 +128,20 @@ - (NSString *)text
 
 - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
 {
+  /**
+   * Only allow single keypresses for onKeyPress, most text pasted in will not be sent.
+   *
+   * Unfortunately, UITextView for some reason doesn't implement `paste:sender` like UITextField, so we have
+   * no way of distinguishing a single character paste from a single character keyPress. :(
+   * I'm assuming this is a bug in iOS, but I couldn't find out for sure.
+   *
+   * End result: Single character pastes will register and fire a keyPress event with the character
+   * that was pasted.
+   */
+  if (text.length <= 1) {
```

Comment:
Ah nice! This was definitely the portion of this feature that I felt bad about not working correctly. Hopefully it works out with his fixes. Thanks.